cpp/test/grpc/wait_callback.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// TODO: move unit_test_framework.h to cpp/test/inc
+#include "../core/unit_test_framework.h"
```

Comment:
Maybe better to adjust the build script to make, let's say `../core`, as an additional include directory?